src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public interface Figure {
+
+    double getArea();
+
+    void draw();
+
+    String getColor();
+}
```

Comment:
Figure should be an abstract class that implements two interfaces you create: one for draw, second for area, then in this class you should add field for color and provide default `getColor()` implementation.